package.json

```diff
@@ -22,6 +22,9 @@
   "dependencies": {
     "@ensdomains/ensjs": "^2.0.1",
     "@interep/reputation": "^0.4.0",
+    "@r1oga/passport-reddit": "^1.0.0",
```

Comment:
This is the package I forked so that it is not bundled as an ES module
https://github.com/r1oga/passport-reddit